store/sqlstore/post_store.go

```diff
@@ -2957,16 +2957,23 @@ func (s *SqlPostStore) updateThreadsFromPosts(transaction *sqlxTxWrapper, posts
 			if err != nil {
 				return err
 			}
+			// get teamId
+			var teamId string
+			err = transaction.Get(&teamId, "SELECT COALESCE(Channels.TeamId, '') FROM Channels WHERE Channels.Id=?", posts[0].ChannelId)
```

Comment:
Let's change this to a sub-query to the root query, instead of a separate query.